rskj-core/src/main/java/co/rsk/config/SyncMode.java

```diff
@@ -0,0 +1,6 @@
+package co.rsk.config;
+
+public enum SyncMode {
+    FULL,
```

Comment:
will it still support lowercase values?

Reply:
Given that the `NodeCliOptions` approach converts the string to uppercase before mapping it to the enum value, it will indeed handle lowercase values as well